examples/oneview_volume.yml

```diff
@@ -31,10 +31,20 @@
           provisioningParameters:
               provisionType: 'Full'
               shareable: True
-              requestedCapacity: 1073741824  # 1GB
+              requestedCapacity: 2147483648  # 2GB
               storagePoolUri: '{{ storage_pool_uri }}'
       delegate_to: localhost
 
+    - name: Extend the Volume size
+      oneview_volume:
+        config: '{{ config }}'
+        state: present
+        data:
+          name: 'Volume with Storage Pool'
+          provisionedCapacity: 2684354560  # 2.5GB
```

Comment:
I get that this increased, but perhaps mention the difference between provisioned and allocated capacity?